pkg/restic/commands.go

```diff
@@ -62,18 +42,27 @@ func initBackup(ctx context.Context, globalOpts *GlobalOptions) ([]byte, error)
 	return out, err
 }
 
-func parseSnapshotOut(str string) (BackupResult, error) {
+func parseSnapshotOut(responses Response) (BackupResult, error) {
```

Comment:
Maybe we can make this function a receiver? `func (r *Response) parseSnapshoutOut() (BackupResult, error)` 

Since it does not need any additional arguments except the response itself.